elementor/elementor
# fix javascript errors in editor when reporting backend deprecations

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fixes frontend errors in the editor when deprecation notices  need to be reported.

## Description
An explanation of what is done in this PR

* the deprecation property was being used in the constructor without being initialised. i moved it so the deoprecation property is set in the constructor before being accessed.

## Test instructions
This PR can be tested by following these steps:

* install a 3rd party plugin that uses deprecated methods or actions from elementor api eg happy-elementor-addon
* edit a page in elementor
* check the console for errors relating to elementorDevTools not being defined

## Quality assurance


Fixes #
